src/views/AgmExtension.vue

```diff
@@ -278,6 +279,11 @@ export default class AgmExtension extends Mixins(CommonMixin, DateMixin, FilingM
 
     // if required data isn't set, go back to dashboard
     if (!this.getIdentifier || this.filingId !== 0) {
+      if (GetFeatureFlag('use-business-dashboard')) {
+        const dashboardUIUrl = `${this.getBusinessDashUrl}/${this.getIdentifier}`
+        navigate(dashboardUIUrl)
```

Comment:
To reduce code duplication (and to show that all 4 places in here do the same thing), I recommend creating a small method that builds the URL and does the navigation. You can have an optional filingId parameter to handle line 395.

Reply:
On second thought, check if all filings use Common Mixin and move that method there maybe?

In fact, that method could look at the FF and determine whether to route or redirect. What do you think?

Reply:
I agree, I will update the Mixin, thanks.